From: Stephen Hemminger <stephen@networkplumber.org>
Date: Thu, 6 Mar 2014 05:28:12 +0000
Subject: [PATCH v3 302/474] i40e,igb,ixgbe: remove usless return statements
Patch-mainline: v3.16-rc1
Git-commit: 41457f64da53112996d2ac607fbb2681e71a9e97 (partial)
References: bsc#909484 FATE#317397

Remove cases where useless bare return is left at end of function.

Signed-off-by: Stephen Hemminger <stephen@networkplumber.org>
Tested-by: Aaron Brown <aaron.f.brown@intel.com>
Tested-by: Kavindya Deegala <kavindya.s.deegala@intel.com>
Tested-by: Phil Schmitt <phillip.j.schmitt@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Acked-by: David Chang <dchang@suse.com>
---
 drivers/net/ethernet/intel/i40e/i40e_main.c |    4 ----
 1 file changed, 4 deletions(-)

--- a/drivers/net/ethernet/intel/i40e/i40e_main.c
+++ b/drivers/net/ethernet/intel/i40e/i40e_main.c
@@ -6828,7 +6828,6 @@ static void i40e_vsi_delete(struct i40e_
 		return;
 
 	i40e_aq_delete_element(&vsi->back->hw, vsi->seid, NULL);
-	return;
 }
 
 /**
@@ -7568,8 +7567,6 @@ void i40e_veb_release(struct i40e_veb *v
 
 	i40e_aq_delete_element(&pf->hw, veb->seid, NULL);
 	i40e_veb_clear(veb);
-
-	return;
 }
 
 /**
@@ -8050,7 +8047,6 @@ static void i40e_determine_queue_usage(s
 	}
 
 	pf->queues_left = queues_left;
-	return;
 }
 
 /**
